### PR TITLE
Fix assertion condition, in IS_SCALAR case.

### PR DESCRIPTION
**Pull request by haradh1**
This is a small fix for install-check failure. Assertion condition is overlooked in the scalar case.

http://jira.madlib.net/browse/MADLIB-304

### CODE REVIEW

File: methods/svec/src/pg_gp/operators.c [hunk 1]
```diff
@@ -241,7 +241,8 @@ Datum svec_change(PG_FUNCTION_ARGS)
 	int midlen = middle->total_value_count;
 	SparseData head = NULL, tail = NULL, ret = NULL;
 
-	Assert(midlen == changed->dimension);
+	Assert((IS_SCALAR(changed) && midlen == 1) ||
```
Comment (fschopp) [inline methods/svec/src/pg_gp/operators.c:244]: I think it is helpful to add a comment to the source code and/or reference the MADlib Jira.

Comment (haradh1) [inline methods/svec/src/pg_gp/operators.c:244]: Well, I think it's obvious without links or comments. Do we need every fix linking to Jira??

Comment (haradh1) [inline methods/svec/src/pg_gp/operators.c:244]: But feel free to modify it when you pull it. Thanks

**Comment by cwelton** [comment]
Merged.

**Pull request merged**